examples/usecase-passport/schema.ts

```diff
@@ -0,0 +1,17 @@
+import { list } from "@keystone-6/core";
+import { allowAll } from "@keystone-6/core/access";
+import { text, password, timestamp } from "@keystone-6/core/fields";
+import type { Lists } from ".keystone/types";
+import { Session } from "./auth";
+
+export const lists = {
```

Comment:
Is it better to make this schema similar to the next-auth example https://github.com/keystonejs/keystone/blob/main/examples/custom-session-next-auth/schema.ts and use `authId` as `p.id` returned from passport?

Reply:
Tried to replicate the schema of next-auth example. Not sure if did it right.